pages/mesosphere/dcos/services/data-science-engine/1.0.1/custom-images/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: layout.pug
+navigationTitle: Custom Images 
+excerpt: Using Custom Images in DC/OS Data Science Engine
+title: Custom Images
+menuWeight: 7
+enterprise: true
+model: /mesosphere/dcos/services/data-science-engine/data.yml
+render: mustache
+---
+
+This guide will show you how to build a custom Docker image for Spark Executor.
```

Comment:
Let's add some explanation why it might be needed e.g. to use custom-built images with libraries used in distributed Spark jobs

Reply:
"This guide will show you how to build a custom Docker image for Spark Executor. You can use these custom-built images with libraries used in distributed Spark jobs."

OK?

Reply:
LGTM 👍